packages/expo-updates/e2e/setup/project.js

```diff
@@ -478,7 +478,7 @@ async function initAsync(
   // enable proguard on Android
   await fs.appendFile(
     path.join(projectRoot, 'android', 'gradle.properties'),
-    '\nandroid.enableProguardInReleaseBuilds=true',
+    '\nandroid.enableProguardInReleaseBuilds=true\nandroid.kotlinVersion=1.8.20',
```

Comment:
given that you're using prebuild here, it would be better to use `expo-build-properties` to set the kotlin version via config plugin: https://docs.expo.dev/versions/latest/sdk/build-properties/#pluginconfigtypeandroid

Reply:
@brentvatne very sadly that does not work -- it seems that the Detox config plugin is getting applied _after_ the `expo-build-properties` plugin. This seems very strange, as my `plugins` array has `expo-build-properties` last.